### PR TITLE
Updating to add our Laravel related style guides and styleci rules.

### PR DESCRIPTION
**Pull request by weerd**
#### What's this PR do?

This PR adds some of our coding style guide related rules and includes a **.styleci.yml** file we can use in our projects to universalize our rules across Laravel repositories.
#### Where should the reviewer start?

Check out the rules. Do we all agree?
#### Any background context you want to provide?

Let's discuss here what rules we want and iron it all out so we have one place to define them all :dancer: 

List of all the [fixers](https://styleci.readme.io/docs/fixers)

List of the [Laravel predefined fixers](https://styleci.readme.io/docs/presets#laravel) to compare which ones we've decided to override.

---

@angaither @chloealee @deadlybutter @DFurnes @katiecrane @sbsmith86

### CODE REVIEW

File: php/laravel/.styleci.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+preset: laravel
+
+linting: true
+
+disabled:
+ - concat_without_spaces
```
Comment (DFurnes) [inline php/laravel/.styleci.yml:6]: :sob:

**Comment by DFurnes** [comment]
I'd also nominate disabling the [`phpdoc_summary` fixer](https://styleci.readme.io/docs/fixers#phpdoc_summary), which checks that all docblocks end with punctuation. It's generally nice, but kinda crummy if you want to do something like include a URL in a docblock like `POST users/` (because it suggest you make it `POST users/.`

**Comment by sbsmith86** [comment]
i'm good with these.

**Comment by weerd** [comment]
@DFurnes I had it in to disable before, and figured it was a nice to have punctuation, but didn't realize it affected things like that @_@ I'll add back!

**Comment by katiecrane** [comment]
These seem fine to me

File: php/laravel/.styleci.yml (hunk 1 above)
Comment (weerd) [inline php/laravel/.styleci.yml:6]: 1925cf0e-eb7f-11e5-998e-098af1dbfc75

**Comment by weerd** [comment]
@DFurnes fixed the summary one ;)

File: php/laravel/.styleci.yml (hunk 1 above)
Comment (DFurnes) [inline php/laravel/.styleci.yml:6]: RUDE. (Seriously though, very low care-o-meter on this so happy to change).

**Comment by DFurnes** [comment]
:+1:

**Pull request merged**